react/react1/week1/todo/src/tasks.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
There's a nice extension to vs code called Prettier that would format everything for you on save. :)

Reply:
Actually, I use prettier the whole time. But somehow it didn't worked on this .json file... maybe something with config...
Formatted it manually this time!